awesomplete.js

```diff
@@ -42,15 +43,18 @@ var _ = function (input, o) {
 
 	this.ul = $.create("ul", {
 		hidden: "hidden",
+        role: "listbox",
+        id: "awescomplete_list",
```

Comment:
Also think I spelt awesomplete wrong in a few names, so will correct that too :)